src/views/tickets/orders.php

```diff
@@ -43,7 +43,7 @@
 
 	<form method="post">
 
-	<?php tribe_tickets_get_template_part( 'tickets/orders-rsvp' ); ?>
+	<?php require( 'orders-rsvp.php' ); ?>
```

Comment:
Without the require only one instance of the template will load.  So an event with both RSVP and paid tickets will only display meta for the RSVP.